deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,682 @@
+    [poolRegistry, "addMarket(AddMarketInput)", AccountType.NORMAL_TIMELOCK],
+    [poolRegistry, "setPoolName(address,string)", AccountType.NORMAL_TIMELOCK],
+    [poolRegistry, "updatePoolMetadata(address,VenusPoolMetaData)", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+const getPrimePermissions = (prime: string): string[][] => {
+  return [
+    accounts.flatMap(timelock => [prime, "updateAlpha(uint128,uint128)", timelock]),
+    accounts.flatMap(timelock => [prime, "updateMultipliers(address,uint256,uint256)", timelock]),
+    accounts.flatMap(timelock => [prime, "setStakedAt(address[],uint256[])", timelock]),
+    accounts.flatMap(timelock => [prime, "addMarket(address,address,uint256,uint256)", timelock]),
+    accounts.flatMap(timelock => [prime, "setLimit(uint256,uint256)", timelock]),
+    accounts.flatMap(timelock => [prime, "setMaxLoopsLimit(uint256)", timelock]),
+    accounts.flatMap(timelock => [prime, "issue(bool,address[])", timelock]),
+    accounts.flatMap(timelock => [prime, "issue(bool,address[])", timelock]),
+    accounts.flatMap(timelock => [prime, "burn(address)", timelock]),
+    accounts.flatMap(timelock => [prime, "togglePause()", timelock]),
+  ];
+};
+
+const getPrimeLiquidityProviderPermissions = (primeLiquidityProvider: string): string[][] => {
+  return [
+    accounts.flatMap(timelock => [primeLiquidityProvider, "setTokensDistributionSpeed(address[],uint256[])", timelock]),
+    accounts.flatMap(timelock => [
+      primeLiquidityProvider,
+      "setMaxTokensDistributionSpeed(address[],uint256[])",
+      timelock,
+    ]),
+    accounts.flatMap(timelock => [primeLiquidityProvider, "setMaxLoopsLimit(uint256)", timelock]),
+    accounts.flatMap(timelock => [primeLiquidityProvider, "pauseFundsTransfer()", timelock]),
+    accounts.flatMap(timelock => [primeLiquidityProvider, "resumeFundsTransfer()", timelock]),
+  ];
+};
+
+const getProtocolShareReservePermissions = (protocolShareReserve: string): string[][] => {
+  return [
+    accounts.flatMap(timelock => [
+      protocolShareReserve,
+      "addOrUpdateDistributionConfigs(DistributionConfig[])",
+      timelock,
+    ]),
+    accounts.flatMap(timelock => [protocolShareReserve, "removeDistributionConfig(Schema,address)", timelock]),
+  ];
+};
+
+const getConverterNetworkPermissions = (converterNetwork: string): string[][] => {
+  return [
+    accounts.flatMap(timelock => [converterNetwork, "addTokenConverter(address)", timelock]),
+    accounts.flatMap(timelock => [converterNetwork, "removeTokenConverter(address)", timelock]),
+  ];
+};
+
+const getComptrollerPermissions = (): string[][] => {
+  return [
+    accounts.flatMap(timelock => [
+      ethers.constants.AddressZero,
+      "setCollateralFactor(address,uint256,uint256)",
+      timelock,
+    ]),
+    accounts.flatMap(timelock => [ethers.constants.AddressZero, "setMarketBorrowCaps(address[],uint256[])", timelock]),
```

Comment:
I miss permissions for `setMarketSupplyCaps(address[],uint256[])`. I think we should authorize every timelock to execute that function too